apex/python/manager.py

```diff
@@ -9,27 +10,44 @@ class EventManager(Publisher):
 
     def __init__(self) -> None:
         super().__init__()
+        self.pub_m = message()
```

Comment:
For convention, "private" members should begin with "_" for example: self._pub_m = message().